templates/_macros/vf_text-spotlight.jinja

```diff
@@ -3,9 +3,14 @@
       @param title_text: H2 title text
       @param list_items (Array<string>)
       An array of HTML strings to be rendered
+      @param item_heading_level: Level of the heading to use, defaults to '2'. Can be '2', or '4'.
 #}
-{%- macro vf_text_spotlight(title_text, list_items=[], caller=None) -%}
+{%- macro vf_text_spotlight(title_text, item_heading_level='2', list_items=[], caller=None) -%}
   {% set has_list = list_items | length >= 2 and list_items | length <= 7 %}
+  {% set item_heading_level = item_heading_level | trim %}
+  {%- if item_heading_level not in ["2", "4"] -%}
+    {%- set item_heading_level = "2" -%}
+  {%- endif -%}
```

Comment:
We have been using integers for these in other patterns, I'd recommend doing the same here.

https://github.com/canonical/vanilla-framework/blob/f47bf62b5e555267e2cd77d8df65f5f0abe84820/templates/_macros/vf_equal-heights.jinja#L30-L32